cloudformation/create-stack.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash -e
+
+set -euo pipefail
+
+stack_creating=
+master_hostname=
+
+function finish {
+    if [ -n "${stack_creating}" ]; then
+        # Print error messages if any
+        echo >&2
+        aws cloudformation describe-stack-events \
+            --stack-name ${stack_name} \
+            --query "StackEvents[?ResourceStatus==\`CREATE_FAILED\`].ResourceStatusReason" \
+            --output text
+
+        echo >&2
+        echo Deleting stack ${stack_name} >&2
+        aws cloudformation delete-stack --stack-name "${stack_name}"
```

Comment:
Also, created [a ticket](https://github.com/citusdata/test-automation/issues/24) for the error you experienced.